src/client/app.json

```diff
@@ -1,19 +0,0 @@
-{
```

Comment:
looking at it it just launches the open fin demo. Its not referenced anywhere so deleting it. When we do that story we can add a suitable version.


Sent from Outlook Mobile






On Thu, Apr 28, 2016 at 1:35 AM -0700, "Dimitar Christoff" <notifications@github.com> wrote:





> @@ -1,19 +0,0 @@
> -{

this app.json is for launching the chartIQ demo, though it may be out of date

---
You are receiving this because you authored the thread.
Reply to this email directly or view it on GitHub:
https://github.com/AdaptiveConsulting/ReactiveTraderCloud/pull/313/files/6904b34c504d71629a7fec8788545940a1bb52fd#r61391421